docs/assets-scripts.md

```diff
@@ -1 +1,193 @@
-Not ready yet. Contribution are welcome!
\ No newline at end of file
+# Add External Scripts
+It is really easy to use any external javascript with gridsome. Being a Vue based framework any method of importing external scripts in vue works out of the box with Gridsome
+
+## Defining external dependencies Globally
+
+### Using an external Vue Plugin 
+To use any external vue plugin with gridsome. Just import the required plugin and pass the required plugin to Vue using the following function `Vue.use` inside your main.js file
+
+Example:
+```javascript
+//main.js 
+
+import DefaultLayout from '~/layouts/Default.vue'
+import VueTypedJs from 'vue-typed-js'
+
+
+export default function (Vue) {
+  // Set default layout as a global component
+  Vue.component('Layout', DefaultLayout)
+  //Use VueTypedJs plugin inside our project
+  Vue.use(VueTypedJs)
```

Comment:
It's a bad practice to make a plugin globally available. Maybe we should just show how to add it to a component? Or move the docs about components here up.